config/root-config.in

```diff
@@ -828,6 +831,7 @@ while test $# -gt 0; do
       echo "  --python-version      Print the Python version used by ROOT"
       echo "  --python2-version     Print the Python2 version used by PyROOT"
       echo "  --python3-version     Print the Python3 version used by PyROOT"
+      echo "  --cxxstandard         Print the C++ standard used to compile ROOT"
```

Comment:
I don't understand the order of the flags here. Nor the order of the flags in the `--help` output. Nor their relation.

But that's not this PR's business... I'd certainly appreciate e.g. an alphabetical order for both in a subsequent PR :-)